src/api/spec/cassettes/Packages/editing_package_files/editing_an_existing_file.yml

```diff
@@ -756,16 +1132,16 @@ http_interactions:
       Connection:
       - close
       Content-Length:
-      - '122'
+      - '309'
     body:
       encoding: UTF-8
       string: |
-        <status code="404">
-          <summary>_service: no such file</summary>
```

Comment:
but you can see here that the 404 was there before - you just changed the order. And this is what puzzles me: tons of your  vcr additions are from 2018. Did you edit that or just fight with vcr madness?

Reply:
Just fighting...